tools/tidy/src/synthesis/RegisterHasNoReset.cpp

```diff
@@ -95,6 +95,14 @@ struct MainVisitor : public TidyVisitor, ASTVisitor<MainVisitor, true, true> {
         if (symbol.drivers().empty())
             return;
 
+        // Skip iterator variables
+        if (symbol.kind == SymbolKind::Iterator)
+            return;
+
+        // Skip common loop iterator variables
+        if (symbol.name == "i" || symbol.name == "j" || symbol.name == "k")
```

Comment:
Please remove this check. The name should not have any bearing on how the check is applied.